services/discourse_translator/google.rb

```diff
@@ -144,11 +144,19 @@ def self.result(url, body)
 
       if response.status != 200
         if body && body["error"]
-          raise TranslatorError.new(body["error"]["message"])
+          ProblemCheckTracker[:translator_error].problem!(
+            details: {
+              provider: "Google",
+              code: body["error"]["code"],
+              message: body["error"]["message"],
+            },
+          )
+          raise ProblemCheckedTranslationError.new(body["error"]["message"])
```

Comment:
I believe we no longer want to raise here. Since this is not something we at Discourse can do anything about. The admin notice will let the admin know. 🙂

Reply:
No, we need to raise it here. 😉 Because if the user manually sends a request, the `raise` here will `render_json_error`

https://github.com/discourse/discourse-translator/blob/add-problem-check-google/plugin.rb#L91-L92

Reply:
Very good. 👍